aws_lambda_powertools/tracing/tracer.py

```diff
@@ -512,8 +516,8 @@ async def async_tasks():
     def _decorate_async_function(
         self,
         method: Callable = None,
-        capture_response: Optional[bool] = None,
-        capture_error: Optional[bool] = None,
+        capture_response: Optional[Union[bool, str]] = None,
+        capture_error: Optional[Union[bool, str]] = None,
```

Comment:
__Question__: Not sure what this change does. Why do we need to add str as a possible value here?

Reply:
It's because of the`resolve_env_choice..` function that can return either bool or str (1, 0)